### ISSUE TITLE
`Duplicate identifier` error appeared for Teams-js 2.10.0

### ISSUE THREAD

**Issue by yiqing-zhao** [OfficeDev/microsoft-teams-library-js#1695]
Hello, I would appreciate if anyone could help me out. After upgrading my project to use teams-js@2.10.0, I met the following error.

Error:
![image](https://user-images.githubusercontent.com/45935381/230320112-f187f494-616e-404c-a473-708d9feb8179.png)
`node_modules/typescript/lib/lib.dom.d.ts:18499:6 - error TS2300: Duplicate identifier 'VideoColorPrimaries'.`

npm version: 9.5.0
node version: 18.15.0

package.json:
```
{
  "name": "test_command",
  "version": "1.0.0",
  "description": "Microsoft Teams Toolkit Command and Response Bot Sample",
  "engines": {
    "node": "14 || 16 || 18"
  },
  "author": "Microsoft",
  "license": "MIT",
  "main": "./lib/index.js",
  "scripts": {
    "build": "tsc --build ",
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "repository": {
    "type": "git",
    "url": "https://github.com"
  },
  "dependencies": {
    "@microsoft/teams-js": "2.10.0"
  },
  "devDependencies": {
    "typescript": "^4.4.4"
  }
}
```

tsconfig:
```
{
  "compilerOptions": {
    "declaration": true,
    "target": "es2017",
    "module": "commonjs",
    "outDir": "./lib",
    "rootDir": "./",
    "sourceMap": true,
    "incremental": true,
    "tsBuildInfoFile": "./lib/.tsbuildinfo",
    "resolveJsonModule": true,
    "esModuleInterop": true,
  }
}
```

Reproduce steps:
1. `npm install`
2. `npm run build`

**Comment by Meghana-MSFT** [comment]
We did run `npm install` and `npm run build` in this sample https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/tab-navbar-menu/ts

We updated to @microsoft/teams-js": "2.10.0" and had node version as 16. It did not give any error; we will try checking with node 18 as well and get back to you. Thank you.

**Comment by yiqing-zhao** [comment]
> We did run `npm install` and `npm run build` in this sample https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/tab-navbar-menu/ts
> 
> We updated to @microsoft/teams-js": "2.10.0" and had node version as 16. It did not give any error; we will try checking with node 18 as well and get back to you. Thank you.

https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/cfd45e9ca28751d72b91e77a3811a666634a5642/samples/tab-navbar-menu/ts/tsconfig.json#L10 
There is a `"skipLibCheck":true` in the `tsconfig.json`, which might make the compiler ignore the duplicate identifiers. In our cases, we wish to find a better solution instead of just skip the check.

**Comment by TrevorJoelHarris** [comment]
@yiqing-zhao We released 2.10.1 of teamsjs about 15 hours ago, and I suspect that may fix the problem for you. Do you mind trying that version and seeing if it works for you?

https://www.npmjs.com/package/@microsoft/teams-js

**Comment by yiqing-zhao** [comment]
@TrevorJoelHarris That version works fine. Many thanks!
